src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -84,6 +85,12 @@ public abstract class AbstractBigQuerySinkConfig extends BigQueryBaseConfig {
     "This value is ignored if the dataset or temporary bucket already exist.")
   protected String location;
 
+  @Name(NAME_JSON_STRING_FIELDS)
```

Comment:
Add details in MD file also with an example for hierarchical fields also

Reply:
**JSON String**: List of fields to be written to BigQuery as a JSON string.
The fields must be of type STRING. To target nested fields, use dot notation.
For example, 'name.first' will target the 'first' field in the 'name' record. (Macro Enabled) 

Use a comma-separated list to specify multiple fields in macro format.
Example: "nestedObject.nestedArray.raw, nestedArray.raw".